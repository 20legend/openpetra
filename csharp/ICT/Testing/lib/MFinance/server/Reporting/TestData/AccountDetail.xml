<?xml version="1.0"?>
<Parameters>
  <Parameter id="xmlfiles" value="eString:Finance/accountdetail.xml,Finance/accountdetailcommon.xml,Finance/finance.xml,common.xml" />
  <Parameter id="currentReport" value="eString:Account Detail" />
  <Parameter id="MaxDisplayColumns" value="eInteger:6" />
  <Parameter id="param_diff_period_i" value="eInteger:0" />
  <Parameter id="param_account_hierarchy_c" value="eString:STANDARD" />
  <Parameter id="param_currency" value="eString:Base" />
  <Parameter id="param_period" value="eBoolean:true" />
  <Parameter id="param_date_checked" value="eBoolean:false" />
  <Parameter id="param_start_period_i" value="eInteger:1" />
  <Parameter id="param_end_period_i" value="eInteger:1" />
  <Parameter id="param_year_i" value="eInteger:0" />
  <Parameter id="param_sortby" value="eString:Account" />
  <Parameter id="param_account_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_account_code_start" value="eString:&quot;0100&quot;" />
  <Parameter id="param_account_code_end" value="eInteger:9800" />
  <Parameter id="param_rgrAccounts" value="eString:AccountRange" />
  <Parameter id="param_cost_centre_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_cost_centre_code_start" value="eString:&quot;0400&quot;" />
  <Parameter id="param_cost_centre_code_end" value="eInteger:9500" />
  <Parameter id="param_rgrCostCentres" value="eString:CostCentreRange" />
  <Parameter id="param_depth" value="eString:standard" />
  <Parameter id="ColumnPosition" column="0" value="eDecimal:4618441417868443648" />
  <Parameter id="ColumnPositionIndented" column="0" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="0" value="eDecimal:4611686018427387904" />
  <Parameter id="ColumnPosition" column="1" value="eDecimal:4620693217682128896" />
  <Parameter id="ColumnPositionIndented" column="1" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="1" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnPosition" column="2" value="eDecimal:4622382067542392832" />
  <Parameter id="ColumnPositionIndented" column="2" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="2" value="eDecimal:4607182418800017408" />
  <Parameter id="ColumnPosition" column="3" value="eDecimal:4622945017495814144" />
  <Parameter id="ColumnPositionIndented" column="3" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="3" value="eDecimal:4619567317775286272" />
  <Parameter id="ColumnPosition" column="4" value="eDecimal:4626041242239631360" />
  <Parameter id="ColumnPositionIndented" column="4" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="4" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnPosition" column="5" value="eDecimal:4626885667169763328" />
  <Parameter id="ColumnPositionIndented" column="5" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="5" value="eDecimal:4613937818241073152" />
  <Parameter id="param_calculation" column="0" value="eString:Debit" />
  <Parameter id="param_calculation" column="1" value="eString:Credit" />
  <Parameter id="param_calculation" column="3" value="eString:Transaction Narrative" />
  <Parameter id="param_calculation" column="4" value="eString:Start Balance" />
  <Parameter id="param_calculation" column="5" value="eString:End Balance" />
  <Parameter id="param_ledger_number_i" value="43" />
  <Parameter id="param_with_analysis_attributes" value="eBoolean:false" />
  <Parameter id="param_quarter" value="eString:false" />
  <Parameter id="param_daterange" value="eString:false" />
</Parameters>