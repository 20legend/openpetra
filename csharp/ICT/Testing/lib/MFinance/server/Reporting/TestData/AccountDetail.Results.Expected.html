<html>
<head>
<meta charset="utf-8">
<title>OpenPetra Reports</title>
<link href="/css/report.css" rel="stylesheet">
</head>

<body>

<div id="head">
	<div class="title1" style='visibility:hidden'>Account Detail with Analysis Attributes</div>
	<div class="title1" style='visibility:hidden'>Account Detail</div>
	<div class="title2">[getLedgerName(43)]</div>
	<div class="descr1"><!-- TODO --></div>
	<div class="descr2"><!-- TODO --></div>
	<div class="descr3"><!-- TODO --></div>
	<div class="period1"><!-- TODO --></div>

	<div class="period2">
		<div style='visibility:hidden'>Reference: All references</div>
		<div style='visibility:hidden'>Reference:  to </div>
		<div style='visibility:hidden'>Analysis Attributes: All Analysis Attributes</div>
		<div style='visibility:hidden'>Analysis Attributes:  to </div>
		<div style='visibility:hidden'>Cost Centre: 10100 To 10500</div>
		<div style='visibility:hidden'>Cost Centres: *NOTUSED*</div>
	</div>
	<div class="period3"><!-- TODO --></div>
</div>





<div id="column_headings" class="row">
	<div class="col-2">Transaction Date</div>
	<div class="col-2">Reference</div>
	<div class="col-4">Narrative</div>
	<div class="col-1">Debit</div>
	<div class="col-1">Credit</div>
	<div class="col-2">End Balance</div>
</div>
<div class="row">
	<div class="col-12">
		<hr>
	</div>
</div>
<div id="content">
	
<div id="acccc0">
		<div class="header row">
			<div class="header1 col-4"><strong>10300-0100</strong></div>
			<div class="header2 col-8"><strong>CAMBODIA, Support Gifts: Local</strong></div>
		</div>
		
		<div id="trans0" class="row">
			<div class="col-2 date">2016-01-01</div>
			<div class="col-2">GB1</div>
			<div class="col-4">GB - Gift Batch 1</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">410.00</div>
		</div><div id="trans1" class="row">
			<div class="col-2 date">2016-01-10</div>
			<div class="col-2">GB10</div>
			<div class="col-4">GB - Gift Batch 10</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">361.00</div>
		</div><div class="footer row">
			<div class="col-2">Net Balance:</div>
			<div class="col-2">&nbsp;</div>
			<div class="col-4">&nbsp;</div>
			<div class="col-1 currency">0.00</div>
			<div class="col-1 currency">771.00</div>
			<div class="col-2 currency">4020.00</div>
		</div>
		<div class="row">
			<div class="col-12">
				<hr>
			</div>
		</div>
	</div><div id="acccc1">
		<div class="header row">
			<div class="header1 col-4"><strong>10400-0100</strong></div>
			<div class="header2 col-8"><strong>EL SALVADOR, Support Gifts: Local</strong></div>
		</div>
		
		<div id="trans2" class="row">
			<div class="col-2 date">2016-01-04</div>
			<div class="col-2">GB4</div>
			<div class="col-4">GB - Gift Batch 4</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">71.00</div>
		</div><div id="trans3" class="row">
			<div class="col-2 date">2016-01-05</div>
			<div class="col-2">GB5</div>
			<div class="col-4">GB - Gift Batch 5</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">212.00</div>
		</div><div id="trans4" class="row">
			<div class="col-2 date">2016-01-10</div>
			<div class="col-2">GB10</div>
			<div class="col-4">GB - Gift Batch 10</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">376.00</div>
		</div><div id="trans5" class="row">
			<div class="col-2 date">2016-01-12</div>
			<div class="col-2">GB12</div>
			<div class="col-4">GB - Gift Batch 12</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">181.00</div>
		</div><div id="trans6" class="row">
			<div class="col-2 date">2016-01-14</div>
			<div class="col-2">GB14</div>
			<div class="col-4">GB - Gift Batch 14</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">21.00</div>
		</div><div id="trans7" class="row">
			<div class="col-2 date">2016-01-16</div>
			<div class="col-2">GB16</div>
			<div class="col-4">GB - Gift Batch 16</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">68.00</div>
		</div><div id="trans8" class="row">
			<div class="col-2 date">2016-01-21</div>
			<div class="col-2">GB21</div>
			<div class="col-4">GB - Gift Batch 21</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">450.00</div>
		</div><div id="trans9" class="row">
			<div class="col-2 date">2016-01-23</div>
			<div class="col-2">GB23</div>
			<div class="col-4">GB - Gift Batch 23</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">25.00</div>
		</div><div id="trans10" class="row">
			<div class="col-2 date">2016-01-25</div>
			<div class="col-2">GB25</div>
			<div class="col-4">GB - Gift Batch 25</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">113.00</div>
		</div><div id="trans11" class="row">
			<div class="col-2 date">2016-01-28</div>
			<div class="col-2">GB28</div>
			<div class="col-4">GB - Gift Batch 28</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">280.00</div>
		</div><div id="trans12" class="row">
			<div class="col-2 date">2016-01-30</div>
			<div class="col-2">GB30</div>
			<div class="col-4">GB - Gift Batch 30</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">172.00</div>
		</div><div class="footer row">
			<div class="col-2">Net Balance:</div>
			<div class="col-2">&nbsp;</div>
			<div class="col-4">&nbsp;</div>
			<div class="col-1 currency">0.00</div>
			<div class="col-1 currency">1969.00</div>
			<div class="col-2 currency">34508.00</div>
		</div>
		<div class="row">
			<div class="col-12">
				<hr>
			</div>
		</div>
	</div><div id="acccc2">
		<div class="header row">
			<div class="header1 col-4"><strong>10500-0100</strong></div>
			<div class="header2 col-8"><strong>TUNISIA, Support Gifts: Local</strong></div>
		</div>
		
		<div id="trans13" class="row">
			<div class="col-2 date">2016-01-01</div>
			<div class="col-2">GB1</div>
			<div class="col-4">GB - Gift Batch 1</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">189.00</div>
		</div><div id="trans14" class="row">
			<div class="col-2 date">2016-01-02</div>
			<div class="col-2">GB2</div>
			<div class="col-4">GB - Gift Batch 2</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">177.00</div>
		</div><div id="trans15" class="row">
			<div class="col-2 date">2016-01-04</div>
			<div class="col-2">GB4</div>
			<div class="col-4">GB - Gift Batch 4</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">184.00</div>
		</div><div id="trans16" class="row">
			<div class="col-2 date">2016-01-05</div>
			<div class="col-2">GB5</div>
			<div class="col-4">GB - Gift Batch 5</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">473.00</div>
		</div><div id="trans17" class="row">
			<div class="col-2 date">2016-01-08</div>
			<div class="col-2">GB8</div>
			<div class="col-4">GB - Gift Batch 8</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">63.00</div>
		</div><div id="trans18" class="row">
			<div class="col-2 date">2016-01-09</div>
			<div class="col-2">GB9</div>
			<div class="col-4">GB - Gift Batch 9</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">323.00</div>
		</div><div id="trans19" class="row">
			<div class="col-2 date">2016-01-15</div>
			<div class="col-2">GB15</div>
			<div class="col-4">GB - Gift Batch 15</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">393.00</div>
		</div><div id="trans20" class="row">
			<div class="col-2 date">2016-01-19</div>
			<div class="col-2">GB19</div>
			<div class="col-4">GB - Gift Batch 19</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">499.00</div>
		</div><div id="trans21" class="row">
			<div class="col-2 date">2016-01-23</div>
			<div class="col-2">GB23</div>
			<div class="col-4">GB - Gift Batch 23</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">314.00</div>
		</div><div id="trans22" class="row">
			<div class="col-2 date">2016-01-26</div>
			<div class="col-2">GB26</div>
			<div class="col-4">GB - Gift Batch 26</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">426.00</div>
		</div><div id="trans23" class="row">
			<div class="col-2 date">2016-01-27</div>
			<div class="col-2">GB27</div>
			<div class="col-4">GB - Gift Batch 27</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">265.00</div>
		</div><div id="trans24" class="row">
			<div class="col-2 date">2016-01-28</div>
			<div class="col-2">GB28</div>
			<div class="col-4">GB - Gift Batch 28</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">470.00</div>
		</div><div id="trans25" class="row">
			<div class="col-2 date">2016-01-29</div>
			<div class="col-2">GB29</div>
			<div class="col-4">GB - Gift Batch 29</div>
			<div class="col-1 currency"></div>
			<div class="col-1 currency">264.00</div>
		</div><div class="footer row">
			<div class="col-2">Net Balance:</div>
			<div class="col-2">&nbsp;</div>
			<div class="col-4">&nbsp;</div>
			<div class="col-1 currency">0.00</div>
			<div class="col-1 currency">4040.00</div>
			<div class="col-2 currency">68700.00</div>
		</div>
		<div class="row">
			<div class="col-12">
				<hr>
			</div>
		</div>
	</div></div>

</body>
</html>
